docs/admin/sharding-partitioning.md

```diff
@@ -92,23 +92,7 @@ INSERT INTO second_table (ts, val) VALUES (1620415701974, 2.31);
 We can see that there are now 8 shards for the table `second_table` in the
 cluster.
 
-:::{danger}
-**Over-sharding and over-partitioning**
-
-Sharding can drastically improve the performance on large datasets.
-However, having too many small shards will most likely degrade performance.
-Over-sharding and over-partitioning are common flaws leading to an overall
-poor performance.
-
-**As a rule of thumb, a single shard should hold somewhere between 5 - 50
-GB of data.**
-
-To avoid oversharding, CrateDB by default limits the number of shards per
-node to 1000. Any operation that would exceed that limit, leads to an
-exception.
-:::
```

Comment:
Why remove this? Isn't it good to have as a top highlighted note? Also it correctly mentions `by default ... 1000 shards` while the text below is missing this `default` part.